robot.py

```diff
@@ -40,7 +50,8 @@ def autonomousInit(self) -> None:
         This method runs the autonomous command selected by your
         RobotContainer class.
         """
-        self.autonomousCommand = self.container.getAutonomousCommand()
+        self.autonomousCommand = self.container.get_autonomous_command()
+        #self.autonomousCommand = self.container.getAutonomousCommand()
```

Comment:
This is an excellent commented-out line to remove.  Someone might not notice the `#` at the beginning of the line, and think that `getAutonomousCommand()` is the method we use, when it's actually `get_autonomous_method()`.

For that matter, I hope we don't actually have two methods named `get_autonomous_command()` and `getAutonomousCommand()` -- that's horribly confusing.